pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumenvoyconfigs.yaml

```diff
@@ -0,0 +1,118 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: ciliumenvoyconfigs.cilium.io
+spec:
+  group: cilium.io
+  names:
+    categories:
+    - cilium
+    kind: CiliumEnvoyConfig
+    listKind: CiliumEnvoyConfigList
+    plural: ciliumenvoyconfigs
+    shortNames:
+    - cec
+    singular: ciliumenvoyconfig
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - description: The age of the identity
+      jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v2alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              backendServices:
+                description: BackendServices specifies Kubernetes services whose backends
+                  are automatically synced to Envoy usign EDS.
+                items:
+                  properties:
+                    name:
+                      description: Name is the name of a destination Kubernetes service
+                        that identifies traffic to be redirected.
+                      type: string
+                    namespace:
+                      description: Namespace is the Kubernetes service namespace.
+                      type: string
+                  required:
+                  - name
+                  - namespace
+                  type: object
+                type: array
+              ingress:
```

Comment:
what about exchanging this boolean for `flowtype` with the options being `ingress` and `egress`? perhaps it would be a little more descriptive.